src/transforms/deployment.ts

```diff
@@ -146,24 +155,48 @@ export function mapEnvironment(environment: string): string {
 	return jiraEnv;
 }
 
-export default async (githubClient: GitHubAPI, payload: WebhookPayloadDeploymentStatus, jiraHost: string, logger: LoggerWithTarget): Promise<JiraDeploymentData | undefined> => {
-	const deployment = payload.deployment;
-	const deployment_status = payload.deployment_status;
+export default async (
+	githubClient: GitHubAPI,
+	payload: WebhookPayloadDeploymentStatus,
+	githubInstallationId: number,
+	jiraHost: string,
+	logger: LoggerWithTarget
+): Promise<JiraDeploymentData | undefined> => {
+	const { deployment, deployment_status, repository } = payload
```

Comment:
Cleaned these up because the repetitive dot notation was driving me crazy (and is messier and harder to read).